todocli/todolist.py

```diff
@@ -0,0 +1,17 @@
+from enum import Enum
+
+
+class TodoList:
+    class WellKnownListName(Enum):
+        none = "none"
+        DefaultList = "defaultList"
+        FlaggedEmails = "flaggedEmails"
```

Comment:
What is this used for?

Reply:
Again, this corresponds with the Graph API (see https://docs.microsoft.com/en-us/graph/api/resources/todotasklist?view=graph-rest-1.0 -> Properties -> wellknownListName).

As this value is returned as a response when querying data from the REST API, I included it in the object so it can be used in the future.

Reply:
Ah I see. OK, depending on the results of the `enum` discussion above, this may have to be changed too.